CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.25)
```

Comment:
Should only need `3.21`, no?

> PLUGIN <plugin>
>
>    Added in version 3.21.
>
>    An optional plugin executable. This could be, for example, the path to grpc_cpp_plugin.

Reply:
We'll need a couple of features from 3.25 pretty soon, namely cmake-presets enhancements and `add_subdirectory(... SYSTEM)`, so I'm bumping it in advance.